src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -32,23 +32,6 @@ public override BoundNode VisitConversion(BoundConversion node)
             var toType = node.Type;
             Debug.Assert(result.Type.Equals(toType, TypeCompareKind.IgnoreDynamicAndTupleNames | TypeCompareKind.IgnoreNullableModifiersForReferenceTypes));
 
-            // 4.1.6 C# spec: To force a value of a floating point type to the exact precision of its type, an explicit cast can be used.
-            // It means that explicit casts to (double) or (float) should be preserved on the node.
-            // If original conversion has become something else with unknown precision, add an explicit identity cast.
-            if (!_inExpressionLambda &&
```

Comment:
Why is this removed?

Reply:
Ah, I see that it was moved. 

---
In reply to: [301354883](https://github.com/dotnet/roslyn/pull/37052#discussion_r301354883) [](ancestors = 301354883)